src/Our.Umbraco.InnerContent/Web/Controllers/InnerContentApiController.cs

```diff
@@ -75,10 +76,17 @@ public IEnumerable<object> GetContentTypesByAlias([ModelBinder] string[] aliases
 
         [HttpGet]
         [UseInternalActionFilter("Umbraco.Web.WebApi.Filters.OutgoingEditorModelEventAttribute", onActionExecuted: true)]
-        public ContentItemDisplay GetContentTypeScaffoldByGuid(Guid guid)
+        public ContentItemDisplay GetContentTypeScaffoldByGuid(Guid contentTypeGuid, int blueprintId = 0)
         {
-            var contentType = Services.ContentTypeService.GetContentType(guid);
-            return new ContentController().GetEmpty(contentType.Alias, -20);
+            var controller = new ContentController();
+
+            if (blueprintId > 0)
+            {
+                return controller.GetEmpty(blueprintId, -20);
```

Comment:
If supplying the blueprintId means the guid isn't needed, I'm wondering if this should mean 2 seperate methods should be present instead GetByGuid and GetByBlueprintId?

Reply:
I'd had a thought that we could try to use a single `Guid` for both ContentType and Blueprint - and the controller method can figure out which is which?

But I'm also happy for additional method to specify ContentType or Blueprint.

Reply:
I think the seperate methods would be more explicit so easier to know what is going on